examples/hello/jest.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
This shouldn't be duplicated in all examples, I expect to be centrally managed

Reply:
The reason this isn't centrally managed vis-a-vis the package tests is because the examples maintain a different structure for where snapshots are located, so they need a separate snapshot resolver. If you want, we can change the directory structure to be dogmatic, but if you want to keep this structure, we need a separate config here.

Reply:
Still not sure why we can't reuse this file across all examples?